src/main/java/core/basesyntax/files/FileWriterImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String data, String filePath) {
+        Path path = Path.of(filePath);
+        try {
+            Files.write(path, data.getBytes(), StandardOpenOption.CREATE,
+                    StandardOpenOption.TRUNCATE_EXISTING);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Don't leave `e.printStackTrace()` or `System.out.println(e.getMessage())` in the catch blocks. Let's rethrow a RuntimeException with an informative message and exception object. For example, `throw new RuntimeException("Can't write data to file: " + filePath, e);`